test/integration/fixtures/diffs/output

```diff
@@ -1,91 +1,91 @@
 // DIFF
-      -foo rar baz
-      +foo bar baz
+      -foo bar baz
```

Comment:
why did this have to happen?

Reply:
It and similar changes because of the second commit which changes the order of `actual` and `expected` in diff generation

Reply:
why did we change the order?

Reply:
It is necessary for more natural message in diff. diff shows the reason for which `actual` does not correspond to `expected`. If the reason that at `actual` there is an excess element, then the reason shall show that this excess element is added, thus, I know that I need to find the place where the excess element is added and not to add it.  Thus, for 
* `expected = [1, 2]`
* `actual = [1]`

diff shall look so:
```diff
 1
-2
```
and for 
* `expected = [2]`
* `actual = [1, 2]`

diff shall look so:
```diff
+1
 2
```

Without this change everything was on the contrary that at least confuses me a little